### DISCUSSION TITLE
[已解决]使用Nginx SNI分流 + REALITY 协议后.可以实现Fullcone吗

### DISCUSSION THREAD

**Discussion by thisdk** [XTLS/Xray-core#1745] [Q&A]
如题.现在服务器是使用REALITY协议使用整个443端口.也没设置回落.

如果我使用Nginx SNI分流后.xray部署在docker的host网络.可以使用Fullcone吗

**Comment by RPRX** [comment]
当然，早在两年前，Xray-core 就是默认全 FullCone 了，参考 https://github.com/XTLS/Xray-core/discussions/252 https://github.com/XTLS/Xray-core/discussions/237
